app/main/views/applications.py

```diff
@@ -76,6 +76,33 @@ def update_application(application_id):
     return jsonify(application=application.serializable), 200
 
 
+@main.route('/applications/<int:application_id>/admin', methods=['PUT'])
```

Comment:
Should this be PATCH to keep it consistent with the `update_application()` above?

Reply:
nope, the api client doesn't have patch